### PR TITLE
Localization: Localize luck indicator text and shiny labels

### PR DESCRIPTION
**Pull request by josericardo-fo**
## What are the changes?

Localization of the luck text and the shiny text that appears OnHover

## Why am I doing these changes?

They weren't localized

## What did change?

There was only an existing localization in the pokemon-info-container for the shiny rarities, so I revamped it.
Added entries for each word and, since the shiny and luck texts were used in 3 different files each, I added them to the `common` locale folder.

## Luck (Starter Menu)
image

## Luck (Shop)
image

## Shiny (Battle)
image

## Shiny (Pokémon Info Container)
image

## Luck and Shiny (Summary)
image

## How to test the changes?

Check every screen that the text appears.

## Checklist
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?
- [X] Have I tested the changes (manually)?
    - [X] Are all unit tests still passing? (`npm run test`)
- [X] Are the changes visual?
  - [X] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/zh_CN/common.ts [hunk 1]
```diff
@@ -2,4 +2,9 @@ import { SimpleTranslationEntries } from "#app/interfaces/locales";
 
 export const common: SimpleTranslationEntries = {
   "start": "开始",
+  "luckIndicator": "Luck:",
+  "shinyOnHover": "Shiny",
```
Comment (40chyan) [inline src/locales/zh_CN/common.ts:6]:
```suggestion
  "luckIndicator": "幸运：",
  "shinyOnHover": "闪光",
```

File: src/locales/ko/common.ts [hunk 2]
```diff
@@ -2,4 +2,9 @@ import { SimpleTranslationEntries } from "#app/interfaces/locales";
 
 export const common: SimpleTranslationEntries = {
   "start": "시작",
+  "luckIndicator": "Luck:",
+  "shinyOnHover": "Shiny",
```
Comment (EnochG1) [inline src/locales/ko/common.ts:6]:
```suggestion
  "luckIndicator": "행운:",
  "shinyOnHover": "색이 다른",
```

**Review by EnochG1 — COMMENTED** [review]
Korean translation

File: src/locales/de/common.ts [hunk 3]
```diff
@@ -2,4 +2,9 @@ import { SimpleTranslationEntries } from "#app/interfaces/locales";
 
 export const common: SimpleTranslationEntries = {
   "start": "Start",
+  "luckIndicator": "Sorte:",
+  "shinyOnHover": "Shiny",
```
Comment (CodeTappert) [inline src/locales/de/common.ts:6]:
```suggestion
  "luckIndicator": "Glück:",
  "shinyOnHover": "Schillernd",
```

File: src/locales/es/common.ts [hunk 4]
```diff
@@ -2,4 +2,9 @@ import { SimpleTranslationEntries } from "#app/interfaces/locales";
 
 export const common: SimpleTranslationEntries = {
   "start": "Start",
+  "luckIndicator": "Luck:",
+  "shinyOnHover": "Shiny",
+  "commonShiny": "Común",
+  "rareShiny": "Raro",
+  "epicShiny": "Épico",
```
Comment (InnocentGameDev) [inline src/locales/es/common.ts:9]:
```suggestion
  "luckIndicator": "Suerte:",
  "shinyOnHover": "Shiny",
  "commonShiny": "Común",
  "rareShiny": "Raro",
  "epicShiny": "Épico",
```

File: src/locales/fr/common.ts [hunk 5]
```diff
@@ -2,4 +2,9 @@ import { SimpleTranslationEntries } from "#app/interfaces/locales";
 
 export const common: SimpleTranslationEntries = {
   "start": "Lancer",
+  "luckIndicator": "Luck:",
+  "shinyOnHover": "Shiny",
```
Comment (Adri1) [inline src/locales/fr/common.ts:6]:
```suggestion
  "luckIndicator": "Chance :",
  "shinyOnHover": "Chromatique",
```

**Review by Adri1 — COMMENTED** [review]
Added French

**Review by Tempo-anon — APPROVED** [review]

**Pull request merged**